__tests__/integration_testing.test.js

```diff
@@ -129,4 +129,41 @@ describe('getComments', () => {
             });
         })
     }); 
+});
+
+describe('postComments', () => {
+    it('should return a 201 status code', () => {
```

Comment:
We generally don't need a separate test for the status code alone, we have duplicated that assertion in the next test.